packages/grid/x-data-grid-generator/src/renderer/renderEditCurrency.tsx

```diff
@@ -1,46 +1,24 @@
 import * as React from 'react';
 import { GridRenderEditCellParams } from '@mui/x-data-grid';
-import Autocomplete from '@mui/material/Autocomplete';
+import Autocomplete, { autocompleteClasses } from '@mui/material/Autocomplete';
 import InputBase from '@mui/material/InputBase';
-import { createStyles, makeStyles } from '@mui/styles';
-import { createTheme } from '@mui/material/styles';
+import Box from '@mui/material/Box';
+import { styled } from '@mui/material/styles';
 import { CURRENCY_OPTIONS } from '../services/static-data';
 
-// ISO 3166-1 alpha-2
-// ⚠️ No support for IE 11
-function countryToFlag(isoCode: string) {
-  return typeof String.fromCodePoint !== 'undefined'
-    ? isoCode
-        .toUpperCase()
-        .replace(/./g, (char) => String.fromCodePoint(char.charCodeAt(0) + 127397))
-    : isoCode;
-}
-
-const defaultTheme = createTheme();
-const useStyles = makeStyles(
-  (theme) =>
-    createStyles({
-      option: {
-        '& > span': {
-          marginRight: 10,
-          fontSize: 18,
-        },
-      },
-      inputRoot: {
-        ...theme.typography.body2,
-        padding: '1px 0',
-        height: '100%',
-        '& input': {
-          padding: '0 16px',
-          height: '100%',
-        },
-      },
-    }),
-  { defaultTheme },
-);
+const StyledAutocomplete = styled(Autocomplete)(({ theme }) => ({
+  [`& .${autocompleteClasses.inputRoot}`]: {
+    ...theme.typography.body2,
+    padding: '1px 0',
+    height: '100%',
+    '& input': {
+      padding: '0 16px',
+      height: '100%',
+    },
+  },
+}));
```

Comment:
Same here

on the preview
![image](https://user-images.githubusercontent.com/45398769/149364564-41449556-314b-4367-b8d2-36a02f235a3e.png)

on the doc
![image](https://user-images.githubusercontent.com/45398769/149364606-57fa5e4f-3724-4a83-9b11-99d8f97c7b68.png)

Reply:
Fixed in 1420d76.